### PR TITLE
Assistant rebuild 89

### PR DESCRIPTION
**Pull request by knirirr**
This begins the work described in https://docs.google.com/presentation/d/1pBbzjaxtwN0gnCLo4ayKs6R3Sszq79ItxNaLv_ocPRg/edit#slide=id.g28493db5cbd_0_43

### CODE REVIEW

File: src/components/Navigation/QuestionPage.vue [hunk 1]
```diff
@@ -195,10 +202,15 @@ export default {
...
     },
-    goBack() {
-      let previous = this.history.pop();
-      this.$router.push({path: "/" + previous});
-    },
+    resultCountColour() {
+      if (this.getFairSharingRecords.length > 10) {
+        return "success";
+      }
+      else if (this.getFairSharingRecords.length > 0 && this.getFairSharingRecords.length < 10) {
```
Comment (prakhyatox) [inline src/components/Navigation/QuestionPage.vue:209]: this.getFairSharingRecords.length <= 10

Comment (knirirr) [inline src/components/Navigation/QuestionPage.vue:209]:
```suggestion
      else if (this.getFairSharingRecords.length > 0 && this.getFairSharingRecords.length <= 10) {
```

**Comment by knirirr** [comment]
At the moment there's a bug in the researcher -> deposit data route where if a user goes through it once the terminology search box searches terminiologies, but if they return to that page it starts searching databases!

**Comment by allysonlister** [comment]
Apologies, just noticed a very small text thing:

Please can we replace "We use 'resource' to mean either standards, databases or polices." with "We use 'resource' as a general term when describing the standards, databases and polices present within FAIRsharing."

Does that make sense?

**Review by allysonlister — APPROVED** [review]
Although I earlier made a minor text alteration request, otherwise this seems to be working as expected, and I'm happy for this to go out :)

**Review by delphinedauga — APPROVED** [review]

**Review by ramGranell — CHANGES_REQUESTED** [review]
There are errors in the query returns for "policy" selecting any of the following fields:
-Exception to data sharing
-Data protection
-Updating of dmp
-Licenses for outputs
-Data Citation
-Data Preservation
-Monitoring of compliance
-Data availability statement
-Metadata data sharing
-Mandated metadata sharing
-Sharing research software
-DMP Development
-Timing of DMP

**Comment by ramGranell** [comment]
I've just performed a search for a policy with "exceptionsToDataSharing = Yes" (Home->Start->Journal Publisher->Research Areas (empty)->Resource Type (Policy)->Exceptions to data sharing= "Yes"):

The graphql query is
"{multiTagFilter(exceptionsToDataSharing:true ){ id name abbreviation doi type registry status description domains{label id definitions iri synonyms inFairsharing } subjects{label id definitions iri synonyms } taxonomies{label } userDefinedTags{label }}}"

And the reply is an empty array:
multiTagFilter	[]

In the DB in the development server there is at least one policy with this value to true: https://preview.fairsharing.org/3397

The same issue happens with:
"Updating of DMP" (the same policy has value to Yes)
"Monitoring of Compliance" (the same policy has value to Yes)
"Mandatory mandated data sharing" required  (the same policy has value to "required")
"Mandatory metadata data sharing" required  (the same policy has value to "required")
"Mandated DMP creation" required  (the same policy has value to "required")
"Timing of DMP" pre-award (the same policy has value to "pre-award")

The rest of the fields are able to find policies with different values for the fields values.

**Comment by ramGranell** [comment]
After updating the DB in the dev server with yesterday backup copy of the DB, the same empty search replies are returned.

**Comment by ramGranell** [comment]
Now it works for all the fields but not

"Mandated metadata sharing"  and there are records with value required, e.g. 
https://preview.fairsharing.org/3397

**Comment by ramGranell** [comment]
> Now it works for all the fields but not
> 
> "Mandated metadata sharing" and there are records with value required, e.g. https://preview.fairsharing.org/3397

The possible error for this field can be that in the query performed that is:

{multiTagFilter(fairsharingRegistry:["policy"]recordType:["journal","society","project","institution","journal_publisher","funder"]status:["ready","in_development"]mandatedMetadataSharing:["required"]){ id name abbreviation doi type registry status description domains{label id definitions iri synonyms inFairsharing } subjects{label id definitions iri synonyms } taxonomies{label } userDefinedTags{label }}}

with the field "mandatedMetadataSharing:["required"])" being the requested field
In the multi_tag_field, there is not this option but "sharing_metadata":
https://github.com/FAIRsharing/FAIRsharing-API/blob/master/app/graphql/queries/multi_tag_filter.rb#L41

that is the same field in the metadata json policy file:
https://github.com/FAIRsharing/FAIRsharing-API/blob/master/lib/record_schemas/policy_schema.json#L48

**Comment by knirirr** [comment]
In that case the query in the assistant probably needs changing. 
It may well be that the field is supposed to be called mandated_metadata_sharing as there's a mandated_data_sharing; in this case we would have to change the schema files, with all the nuisance that goes with doing that. 
@allysonlister @delphinedauga what are your thoughts on the name of this field?

**Comment by ramGranell** [comment]
I don't think there is need to change the name of field in the .json file,
you can see in the record https://preview.fairsharing.org/3397 as sharing "sharing metadata" in the same way as there is "sharing data"

**Comment by knirirr** [comment]
> ... you can see in the record https://preview.fairsharing.org/3397 as sharing "sharing metadata" in the same way as there is "sharing data"

Indeed, but the structure is different which may seem inconsistent. Anyway, the query should now work.

**Review by ramGranell — APPROVED** [review]

**Review by prakhyatox — APPROVED** [review]

**Pull request merged**